src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,47 @@
+.watch {
+  position: relative;
+  @include makeCircle($watchSize);
```

Comment:
Better place all mixins in the top of the block:
```
.watch {
  @include makeCircle($watchSize);

  position: relative;
  // other rules
```
It's not critical but will be more convenient.